src/components/modals/TransferUnclaimedTokensModal.tsx

```diff
@@ -123,6 +123,7 @@ export function TransferUnclaimedTokensModal({
           rules={[
             {
               validator: validateAmount,
+              validateTrigger: 'onBlur',
```

Comment:
seems like onBlur() shows an error the entire time while typing

example:
 - type invalid amount, no error
 - blur input, error shows up
 - re-focus input, typing. error shows until blur again
 
 maybe we use `'onChange'`, or `['onFocus' | 'onBlur']`?

Reply:
problem is I think onChange is too aggressive, and sometimes doesn't work as intended.

It looks like onBlur isn't working, maybe it should be onSubmit which I think should be fine for this case